main.py

```diff
@@ -0,0 +1,97 @@
+from fastapi import FastAPI, Depends
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.post("/authors/", response_model=schemas.Authors)
```

Comment:
The response model for the `create_author` endpoint should be `schemas.Author` instead of `schemas.Authors` to match the Pydantic model naming convention.